athena-postgresql/src/main/java/com/amazonaws/athena/connectors/postgresql/PostGreSqlMetadataHandler.java

```diff
@@ -420,7 +420,17 @@ private List<TableName> getMaterializedViews(Connection connection, String datab
         return JDBCUtil.getTableMetadata(preparedStatement, MATERIALIZED_VIEWS);
     }
 
-    protected PreparedStatement getMaterializedViewCaseInsensitive(Connection connection, String matviewname, String databaseName) throws SQLException
+    /**
+     * Returns Materialized View for Postgresql Or External Tables for Redshift
+     * Note: Redshift maintain Materialized View in the normal schema metadata as regular tables;
+     *       however maintains External Tables in a separate metadata tables
+     * @param connection
+     * @param matviewname
+     * @param databaseName
+     * @return Prepared Statement
+     * @throws SQLException
+     */
+    protected PreparedStatement getMaterializedViewOrExternalTable(Connection connection, String matviewname, String databaseName) throws SQLException
```

Comment:
Is it still case insensitive?

Reply:
Yes

Reply:
Shortened the name for sake of ease of read